item-impl/src/main/java/com/example/auction/item/impl/PItemEntity.java

```diff
@@ -103,14 +108,26 @@ private Behavior cancelled(PItemState state) {
         builder.setReadOnlyCommandHandler(GetItem.class, this::getItem);
 
         // Ignore these commands, they may come due to at least once messaging
-        builder.setReadOnlyCommandHandler(UpdatePrice.class,
-                (cmd, ctx) -> ctx.reply(Done.getInstance()));
-        builder.setReadOnlyCommandHandler(FinishAuction.class,
-                (cmd, ctx) -> ctx.reply(Done.getInstance()));
+        builder.setReadOnlyCommandHandler(UpdatePrice.class, this::alreadyDone);
+        builder.setReadOnlyCommandHandler(FinishAuction.class, this::alreadyDone);
 
         return builder.build();
     }
 
+    /**
+     * Convenience method to handle when a command has already been processed (idempotent processing).
+     */
+    private void alreadyDone(Object command, ReadOnlyCommandContext<Done> ctx) {
```

Comment:
I'm not sure I like `ignore` (in hindsight I don't like `alreadyDone`either :) ). While there's no processing happening, the method is side-effecting a `Done.getInstance()` back through the `ctx`. 

Overload `done` ? `doNothing` ? `identity` ?

I like the idea of promoting the method into `PersistentEntity` better than adding a `ctx.done` (see your previous comment.